### ISSUE TITLE
[YCQL] version of python six distributed in release package incompatible with Python 3.10+

### ISSUE THREAD

**Issue by saurik** [yugabyte/yugabyte-db#25184]
### Description

Downloading the current version of Yugabyte (2.23.1.0-b220) for Linux (at least) includes an embedded copy of Python six version 1.12.0, which doesn't contain fixes required to run on Python versions 3.10+, causing the following error message:

```
# ycqlsh
Traceback (most recent call last):
  File "/home/saurik/bin/yugabyte-2.23.1.0/bin/ycqlsh.py", line 147, in <module>
    from six.moves import configparser, input
ModuleNotFoundError: No module named 'six.moves'
```

This would be fixed by upgrading to a newer version of python six (such as 1.16.0, which seems to be the version of six that is referenced in the requirements_frozen.txt in the code repository; maybe this is just an issue flushing the build system?).

### Issue Type

kind/bug

### Warning: Please confirm that this issue does not contain any sensitive information

- [X] I confirm this issue does not contain any sensitive information.

**Comment by saurik** [comment]
@ddorian Yes: it still happens with the 2024.2 release. This is to be expected, as I told you exactly what the issue is, and it was not addressed: you are still shipping a very old build of six (version 1.12.0) which doesn't work with newer versions of Python.

```
~/yugabyte-2024.2.0.0$ ls -la lib/six*
-rw-r--r-- 1 113 991 10586 Apr  8  2024 lib/six-1.12.0-py2.py3-none-any.zip
```

**Comment by ddorian** [comment]
ycqlsh cli is based on `cqlsh`. Seems like we're inheriting this https://issues.apache.org/jira/browse/CASSANDRA-19206

**Comment by Extarys** [comment]
```
pip install cassandra-driver==3.29.3
rm -f /opt/yugabyte-<version>/lib/cassandra-driver-*.zip
```
Did it for me - finally :)
